### DISCUSSION TITLE
Build Warning: ` WARN  partial [assets/icon.html] - Cannot find icon: assets/svgs/%!!(MISSING)s(<nil>)/fw.svg`

### DISCUSSION THREAD

**Discussion by anoduck** [gethinode/hinode#1176] [Help] [answered]
Build error was circumvented and reduced to a build warning by enabling the option for `mod-fontawesome` to ignore missing icons. Several attempts were made to discover the source of this error, but it has remained elusive. So, thought it would be beneficial to provide some report of the event.

Can provide more information or examples if needed.

**Comment by markdumay** [comment]
That appears to be a bug. Somehow the `icon.html` partial is called without the required arguments. The `icon.html` partial should validate the arguments before processing them too. Could you share more information indeed, e.g. [submit a bug report](https://github.com/gethinode/hinode/issues/new/choose)? If you're not sure how to reproduce the error, could you perhaps give me (temporary) read access to your repo?

The warning message originates here: https://github.com/gethinode/mod-fontawesome/blob/8f320bd60c2c3d3c739db318692edd7c3957ebd5/layouts/partials/assets/icon.html#L76

**Reply by anoduck** [reply]
@markdumay I will fill out a bug report, and generate a fork from my project repo. This way, I can't unintentionally do anything that would cause the bug to go away. It's probably the result of a corrupted character space, numerous content files were copied over from obsidian.

**Comment by markdumay** [comment, marked as answer]
Thanks for sharing the repo. I've found the issue. The issue is caused by the following content in `content/evidence/financials.md`:

```markdown
> [!info] What are Legal Fines?
> Legal fines are penalties for breaking the law.
```

`[!info]` is not a valid alert type, changing it to `[!note]` will fix the issue. You can check the [available alert types](https://gethinode.com/docs/content/typography/#github-style-alerts) in the docs.

The current warning / error message given by Hinode is not very helpful, I'll see if I can improve the code that renders the alert.

**Reply by markdumay** [reply]
I've released a fix in [v0.27.0-alpha5](https://github.com/gethinode/hinode/releases/tag/v0.27.0-alpha5)

**Reply by anoduck** [reply]
@markdumay That quick? My good Sir, you are faster than Mercury himself, and are the embodiment of lightening.

**Reply by anoduck** [reply]
Confirmed. Fixed.
